compiler/luci-interpreter/src/kernels/Concatenation.cpp

```diff
@@ -69,11 +69,26 @@ void Concatenation::configure()
   Shape output_shape = t0->shape();
   output_shape.dim(axis) = sum_axis;
 
-  // TODO S8 type needs more checking: quantization parameters of all input tensors and the output
-  //  tensor should be the same. Note that there is no such requirement for U8 type.
-  if (t0->element_type() == DataType::S8)
-    throw std::runtime_error("Unsupported type.");
-
+  // If input tensors are INT8 type then quantization parameters of all input tensors and the output
+  // should be the same
+  for (auto current_tensor : _inputs)
+  {
+    if (current_tensor->element_type() == DataType::S8)
+    {
+      LUCI_INTERPRETER_CHECK(current_tensor->quantized_dimension() ==
+                             output()->quantized_dimension());
+
+      auto size = current_tensor->zero_points().size();
+      LUCI_INTERPRETER_CHECK(size == current_tensor->scales().size());
+      LUCI_INTERPRETER_CHECK(size == output()->zero_points().size());
+      LUCI_INTERPRETER_CHECK(size == output()->scales().size());
+      for (size_t i = 0; i < size; ++i)
+      {
+        LUCI_INTERPRETER_CHECK(current_tensor->zero_points()[i] == output()->zero_points()[i]);
+        LUCI_INTERPRETER_CHECK(current_tensor->scales()[i] == output()->scales()[i]);
+      }
```

Comment:
(optional) AFAIK `std::tensor` has overloaded `operator==`.

```suggestion
      LUCI_INTERPRETER_CHECK(current_tensor->zero_points().size() == current_tensor->scales().size());
      LUCI_INTERPRETER_CHECK(current_tensor->zero_points() == output()->zero_points());
      LUCI_INTERPRETER_CHECK(current_tensor->scales() == output()->scales());
```

Reply:
Neat!
I totally forgot about this feature =O

Reply:
Done